backend/core/src/auth/services/user.service.ts

```diff
@@ -172,6 +173,10 @@ export class UserService {
     user.dob = dto.dob
     user.email = dto.email
     user.language = dto.language
+    // if coming from partners dto.jurisdictions can be set
+    user.jurisdictions = dto.jurisdictions
```

Comment:
@seanmalbert Could you explain a bit more the reasoning behind this?

Reply:
@pbn4 , As part of the user management flow, a user can belong to more than one jurisdiction and, given they have permissions, when they create a user they will have to select which jurisdiction they belong to. Therefore it would be set on the incoming dto, not from the request header (which partner's doesn't currently use).